javascript/javascript1/week3/cactus_io.js

```diff
@@ -0,0 +1,128 @@
+const smartPhoneActivities = [];
+const usageLimit = 120;
+
+function addActivity(activity, duration, activities) {
+    const errorMessages = [];
+    const errorMessage = [
+        "ACTIVITY MUST BE A STRING!",
+        "DURATION MUST BE A NUMBER!",
+    ]
+
+    if (typeof activity !== "string") {
```

Comment:
I really like this error checking! Later on this will be super important! 🤩

Reply:
1 tip on how this can be improved:
```suggestion
    if (typeof activity !== "string") {
      console.log('ACTIVITY MUST BE A STRING!');
      return;
    }
    
    if (typeof duration !== "number") {
      console.log('DURATION MUST BE A NUMBER!');
      return;
    } 
```

And if you add the `return` in these if statements, you can delete this:
```
if (errorMessages.length > 0) {
        console.log(errorMessages);
        return;
    }
```

Reply:
Thanks! Actually, my first version looked exactly like yours but then I moved the error messages to an array hahaha. I was trying to follow Rasmus' suggestion on keeping console.log statements separated as much as possible from functions. In what cases do you think this approach is much more useful?